src/client_backend/client_backend.cc

```diff
@@ -337,16 +341,19 @@ ClientBackend::RegisterSystemSharedMemory(
       pa::GENERIC_ERROR);
 }
 
+#ifdef TRITON_ENABLE_GPU
```

Comment:
Wrap all external usages of CUDARuntimeLibraryManager type in `#ifdef TRITON_ENABLE_GPU`